space-provider/kcp/kcp.go

```diff
@@ -235,10 +263,19 @@ func buildRawConfig(baseRaw api.Config, spaceName string) api.Config {
 	contexts := make(map[string]*api.Context)
 	contexts[main] = baseRaw.Contexts[main]
 	// modify server path
-	baseRaw.Clusters[main].Server = strings.ReplaceAll(baseRaw.Clusters[main].Server, main, main+":"+spaceName)
+	baseRaw.Clusters[main].Server = strings.ReplaceAll(baseRaw.Clusters[main].Server, main, spaceName)
```

Comment:
Does the spaceName at this point contains the parent?  Should it?

Reply:
Good question. I will change the code to support both cases here.

Reply:
Done